### DISCUSSION TITLE
Alist Helper | Use Your Alist on Desktop with Ease | 在桌面上轻松管理您的Alist

### DISCUSSION THREAD

**Discussion by Xmarmalade** [AlistGo/alist#4278] [Show and tell]
Alist Helper is a software application developed using Flutter, designed to simplify the use of the desktop version of alist. It can manage alist, allowing you to easily start and stop the alist program.

Alist Helper includes several useful features:

- Automatic launching of alist
- Minimizing to the system tray
- Automatic startup on boot, with the option for silent startup
- Quick access to alist version and administrator information
- Adjustable alist startup parameters. You can customize the startup parameters to meet your specific needs and preferences.

Free. No tracking. No ads.

Currently, this app is available on Windows. Adaptation plans for more platforms are in progress.

Please note that this program does not include the binary files for alist. You will need to download them manually.

Alist Helper是一款使用Flutter开发的软件应用程序，旨在简化桌面版alist的使用。它可以管理alist，让您更轻松地开启、关闭alist程序。

Alist Helper包括多个实用功能
 - 自动启动alist
 - 最小化至系统托盘
 - 开机自启和开机静默启动
 - 能够快速查看alist的版本和管理员信息
 - 可调整的alist启动参数。你可以可以根据自己的特定需求和偏好来自定义启动参数。

免费。无跟踪。无广告。

目前，此应用可在 Windows 上使用。更多平台的适配计划正在进行中。

特别注意，本程序不包含alist的二进制文件，您需要手动下载。

## 屏幕截图 
![image](https://user-images.githubusercontent.com/16839488/235334916-461aba54-3c18-4d0e-bff8-1b3f4fd91556.png)
![image](https://user-images.githubusercontent.com/16839488/235334924-8c53ef83-8452-42e7-b48f-7920007d872f.png)

**Comment by xiaowang7212** [comment]
你这做的比作者都好，作者的桌面版没任何功能· · ·

**Comment by xiaowang7212** [comment]
中文呢 · 哪去啦·

**Reply by Xmarmalade** [reply]
i18n会在后续版本加入

**Reply by Xmarmalade** [reply]
> 中文呢 · 哪去啦·

你好，v0.0.2已经添加了多语言，可否点个star呢？https://github.com/iiijam/alisthelper/releases/tag/v0.0.2

**Reply by xiaowang7212** [reply]
厉害，这效率 必须给个star

**Reply by axdlee** [reply]
mac支持不好，闪退

**Reply by Xmarmalade** [reply]
是因为“自动安装”闪退吗，这个功能未经macOS实际测试，不要使用

**Reply by cactuslll** [reply]
> 是因为“自动安装”闪退吗，这个功能未经macOS实际测试，不要使用

请问，因为自动安装闪退了，有解决办法吗

**Comment by zhwilson1994** [comment]
怎么把多个目录分开挂载成多个盘符？

**Reply by Xmarmalade** [reply]
不支持，如有强烈需求建议支持官方购买alist desktop

**Reply by zhwilson1994** [reply]
官方的已经买了，您做的也不错。

**Comment by VOfficial404** [comment]
高技术力，赞

**Comment by 2413181638** [comment]
求mac版本，老作者的根本打不开，同意了隐私也打不开
